Stylesheet/css/utilities.css

```diff
@@ -0,0 +1,168 @@
+/* Border */
+
+.border2 {
+    border: 2px solid;
+}
+
+/* Border color */
+
+.border-black {
+    border-color: black;
+}
+
+.border-white {
+    border-color: white;
+}
+
+/* Padding */
+
+/* Gap */
+
+/* Margin */
+
+/* Display */
+
+.dis-block {
+    display: block;
+}
+
+.dis-flex {
+    display: flex;
+}
+
+.dis-grid {
+    display: grid;
+}
+
+.dis-inline-block {
+    display: inline-block;
+}
+
+.dis-inline {
+    display: inline;
+}
+
+.dis-inline-flex {
+    display: inline-flex;
+}
+
+.dis-hide {
+    display: none;
+}
+
+/* Positions */
+
+.pos-absolute {
+    position: absolute;
+}
+
+.pos-relative {
+    position: relative;
+}
+
+.pos-fixed {
+    position: fixed;
+}
+
+.pos-sticky {
+    position: sticky;
+}
+
+/* Alignments */
+
+.text-left {
+    text-align: left;
+}
+
+.text-right {
+    text-align: right;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.centered {
```

Comment:
Why are you using absoulte + transform for centering content?

Better approach would be to use flex box

```
display: flex;
align-items:center;
justify-content: center;
```

Reply:
Okay will change this

Reply:
Okay will change this